### PR TITLE
Fixes #306 Add ChinaSo support

### LINKED ISSUE #306: Add ChinaSo support
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

**Expected behavior:**

Add [ChinaSo](http://www.chinaso.com/) support

**Steps to reproduce:**

**Yes, I would like to work on this issue**

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #306 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added ChinaSo support

@vaibhavsingh97 @gabru-md @mariobehling 
Providing heroku deployment at https://murmuring-headland-49787.herokuapp.com/
Please review. Thanks !!

### CODE REVIEW

File: app/scrapers/chinaso.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class ChinaSo(Scraper):
+    """Scrapper class for Ask"""
```
Comment (realslimshanky) [inline app/scrapers/chinaso.py]: It should read "Scrapper class for ChinaSo"

**Comment by bhaveshAn** [comment]
@realslimshanky Please review

**Comment by cclauss** [comment]
Seems to be a duplicate of #309 @bhaveshAn @starlord1311 Can you two please decide which PR we should close and which we should merge?

**Comment by mariobehling** [comment]
Currently we need to focus on performance and have low priority for more search engines. Therefore closing it.

**Pull request closed without merging**